module/VuFind/src/VuFind/RecordDriver/DefaultRecord.php

```diff
@@ -1211,7 +1211,14 @@ public function getSeries()
      */
     public function getShortTitle()
     {
-        return $this->fields['title_short'] ?? '';
+        // Faking an example of HTML that would not be escaped
+        $title = $this->fields['title_short'] ?? '';
+        $words = explode(' ', $title);
+        if (count($words) > 3) {
+            $words[2] = '<em>' . $words[2] . '</em>';
```

Comment:
Just so the effect is easily demoed, on any record with a reasonably long title.